core/src/test/java/org/apache/accumulo/core/data/ResourceGroupIdTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.data;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class ResourceGroupIdTest {
+  @Test
+  public void testIllegalIds() {
+    assertThrows(IllegalArgumentException.class, () -> ResourceGroupId.of(""));
+    assertThrows(IllegalArgumentException.class, () -> ResourceGroupId.of(" "));
+    assertThrows(IllegalArgumentException.class, () -> ResourceGroupId.of("\t"));
```

Comment:
A resource group name of `_` should probably not be valid.
```suggestion
    assertThrows(IllegalArgumentException.class, () -> ResourceGroupId.of("\t"));
    assertThrows(IllegalArgumentException.class, () -> ResourceGroupId.of("_"));
```